### PR TITLE
Add fuature to add two intergers and add route

### PR DESCRIPTION
**Pull request by kemeirnld2**

### CODE REVIEW

File: add_app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from flask import Blueprint
+
+addition_bp =Blueprint('addition_bp',__name__)
+
+from . import  routes
```
Comment (maxwelbarno) [inline add_app/__init__.py]: Please add a space after every file

File: add_app/__pycache__/routes.cpython-310.pyc
Comment (maxwelbarno) [inline add_app/__pycache__/routes.cpython-310.pyc:1]: Please add these to a `.gitignore file`. Create one if you don't have

File: add_app/__init__.py (hunk 1 above)
Comment (kemeirnld2) [reply add_app/__init__.py]: WIP

File: add_app/__pycache__/routes.cpython-310.pyc
Comment (kemeirnld2) [reply add_app/__pycache__/routes.cpython-310.pyc:1]: WIP

File: add_app/__init__.py (hunk 1 above)
Comment (kemeirnld2) [reply add_app/__init__.py]: @codetaskmaster kindly review my codes once again, I have implemented the changes. Thanks

File: add_app/routes.py
Comment (maxwelbarno) [inline add_app/routes.py:1]: Please check on the formatting

File: add_app/__pycache__/__init__.cpython-310.pyc
Comment (maxwelbarno) [inline add_app/__pycache__/__init__.cpython-310.pyc:1]: @kemeirnld2 please remove these files

**Review by maxwelbarno — CHANGES_REQUESTED** [review]

File: add_app/routes.py
Comment (kemeirnld2) [reply add_app/routes.py:1]: WIP

File: add_app/__pycache__/__init__.cpython-310.pyc
Comment (kemeirnld2) [reply add_app/__pycache__/__init__.cpython-310.pyc:1]: WIP

File: add_app/routes.py
Comment (maxwelbarno) [reply add_app/routes.py:1]: Ignore this

**Review by maxwelbarno — APPROVED** [review]

**Pull request merged**